### PR TITLE
Completed css-layout-classes - Chris Mendoza

### PR DESCRIPTION
**Pull request by CTMendoza**
css-layout-classes

### CODE REVIEW

File: css-layout-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  display: flex;
+  flex-direction: column;
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+}
+
+.column-two-third {
+  flex-basis: 75%;
```
Comment (BrettAlbright) [inline css-layout-classes/styles.css]: this class should have a width of `66%` to more closely match the name of the class

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
small update needed

File: css-layout-classes/styles.css (hunk 1 above)
Comment (CTMendoza) [reply css-layout-classes/styles.css]: Has been updated

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**